src/components/calcite-combobox/resources.ts

```diff
@@ -0,0 +1,5 @@
+export const ComboboxAncestorSelector = "calcite-combobox-item-group, calcite-combobox-item";
+export const ComboboxItem = "calcite-combobox-item";
+export const ComboboxItemGroup = "calcite-combobox-item-group";
```

Comment:
If these constants aren't used anywhere else, they could be inlined.

Reply:
I'd like to keep them here since they are used more than once.